### PR TITLE
Update login credentials for docker cloud app.

### PR DESCRIPTION
**Pull request by uhofemei**
Update login credentials for docker cloud app.
Signed-off-by: Ulf Hofemeier <ulf.hofemeier@intel.com>

### CODE REVIEW

File: smarthome-web-portal/tools/docker/README.md [hunk 1]
```diff
@@ -47,12 +47,13 @@ Here are the steps to run the Smart Home Web Portal in a Docker container.
 
 5. Point the portal to the gateway server
    * Open the admin portal thru `http://<host-ip-addr>:3031`
-   * Enter the default login credentials: admin/admin
+   * Enter the default login credentials: dev/P@ssw0rd
```
Comment (gvancuts) [inline smarthome-web-portal/tools/docker/README.md]: That's not correct. The credentials for the Admin portal are still admin/admin.

**Comment by uhofemei** [comment]
Done.

**Comment by uhofemei** [comment]
Done.

File: smarthome-web-portal/tools/docker/README.md [hunk 2]
```diff
@@ -52,7 +52,8 @@ Here are the steps to run the Smart Home Web Portal in a Docker container.
       ![admin portal](../../screenshots/smarthome-adminportal.PNG)
    * Restart the container (Refer to step 6 on how to get the container id)
       `docker restart <container id>`
-   * Login to the Home portal thru `http://<host-ip-addr>:3030` (login credentials: ostro/ostro)
+   * Login to the Home portal thru `http://<host-ip-addr>:3030` (login credentials are:
+   * OTC/!otivity and dev/P@ssw0rd
```
Comment (gvancuts) [inline smarthome-web-portal/tools/docker/README.md]: The formatting is messed  up now and the closing parenthesis is missing.

**Pull request merged**